spacewalk/certs-tools/spacewalk-certs-tools.spec

```diff
@@ -84,13 +86,16 @@ Spacewalk.
 %package -n python2-%{name}
 Summary:        Spacewalk SSL Key/Cert Tool
 Group:          Applications/Internet
+BuildRequires:  (python or python2)
 Requires:       %{name} = %{version}-%{release}
 Requires:       python2-rhn-client-tools
 Requires:       python2-uyuni-common-libs
-%if 0%{?rhel} && 0%{?rhel} <= 5
-Requires:       python-hashlib
-%endif
+%if 0%{?suse_version} || 0%{?rhel}
+BuildRequires:  python2
```

Comment:
Seems we have no this BuildRequires 2 times.

In Line 89 you added `BuildRequires:  (python or python2)` and here we have it again with using macros.
Or should be be just a `Requires` ?

Reply:
True, I forgot to remove line 89. This took me a long time to sort out and identify the issue.
It seems that if you just require "python" on Leap, it replaces it with a preferred version and therefore never installs python 2 in my test case. So line 89 unfortunately will not work for python2 as a simplification.
I will remove line 89.